### DISCUSSION TITLE
Use WebDAV (in this case: DAVx5) as backup target on GrapheneOS

### DISCUSSION THREAD

**Discussion by SHA-NDY** [seedvault-app/seedvault#494] [Q&A]
Hi,
I set up Seedvault on GrapheneOS to use my nextcloud as backup target. This works rather poor as the backup process always gets interrupted and never completes a backup. Backups on USB work fine.
I saw a post somewhere (don't remember where) on CalyxOS someone set up a WebDAV connection via the app DAVx5 to a nextcloud server and using it as backup target. When I configure a WebDAV connection in DAVx5 it works well in my file explorer but doesn't show up in the targets list in Seedvault. Is that something that has to be enabled before compiling the OS or is it just a setting within Seedvault itself?

Sorry for asking on potentially OS related behaviours here.

Thanks!

**Comment by grote** [comment]
> When I configure a WebDAV connection in DAVx5 it works well in my file explorer but doesn't show up in the targets list in Seedvault. 

I think that might be due to the outdated version shipped by your ROM. See https://github.com/seedvault-app/seedvault#requirements

Or maybe they disabled DavX5 intentionally?

**Reply by SHA-NDY** [reply]
My ROM is Android 13 based. Maybe it was disabled intentionally, that's right.

**Comment by mannp** [comment]
I have the same issue using android 11 (11.2-3 seedvault) and the latest davx app, built for lineageos with robotnix.

Is there a minimum version of seedvault that supports davx5 mounts?

The nextcloud app shows up as a destination, but has been sketchy at best, so I have stopped using it.

Thanks.
